scripts/check_enterprise_imports.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This file checks all our AGPL licensed source files to be sure they don't
+# import any enterprise licensed packages (the inverse is fine).
+
+set -euo pipefail
+# shellcheck source=scripts/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
+cdroot
+
+set +e
+find . -regex ".*\.go" | grep -v "./enterprise" | xargs grep -n "github.com/coder/coder/enterprise"
```

Comment:
```suggestion
find . -name ./enterprise -prune -o -name '*.go' -print0 | xargs -0 grep -n "github.com/coder/coder/enterprise"
```

Not at a computer; so can’t test. But use if you want. Silly tool optimization. This even handles spaces in names which isn’t ever going to be a problem with Go source files 🤪